### PR TITLE
survey results + openhumans for mailing

### PR DESCRIPTION
**Pull request by gedankenstuecke**
Should be ready to go. Last time we mailed survey-things via `sudo docker exec sidekiq bundle exec rake survey:send` from the `sidekiq`-host`.

### CODE REVIEW

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -81,7 +81,7 @@ def survey(user)
 													password: ENV.fetch('SURVEY_EMAIL_PASSWORD'),
 													address: ENV.fetch('SURVEY_EMAIL_ADDRESS'),
 													port: ENV.fetch('SURVEY_EMAIL_PORT') }
-		mail(:subject => "openSNP: A one-question survey on height", :to => @user.email,:from => "survey@opensnp.org",delivery_method_options: delivery_options)
+		mail(:subject => "openSNP: Read our survey results and meet Open Humans", :to => @user.email,:from => "survey@opensnp.org" ,delivery_method_options: delivery_options)
```
Comment (tsujigiri) [inline app/mailers/user_mailer.rb]: 😢

Comment (gedankenstuecke) [inline app/mailers/user_mailer.rb]: I know 😢  It's on the mozsprint list, see #301. Btw. will you join in? 😄

Comment (tsujigiri) [inline app/mailers/user_mailer.rb]: Do I need time for that? :smile:

Comment (gedankenstuecke) [inline app/mailers/user_mailer.rb]: Some time would be cool. @philippbayer and I wanted to work on it a lot during those two days if possible. And hopefully other people chime in as well!

**Pull request merged**